README.md

```diff
@@ -1,9 +1,17 @@
-A simple leaderboard app built with Next.js and Tailwind CSS to list top contributors of a GitHub organization.
+A simple leaderboard app built with Next.js and Tailwind CSS to list the top contributors of a GitHub organization.
 
 <img width="1822" alt="image" src="https://github.com/coronasafe/leaderboard/assets/25143503/6352a4cf-4b8b-4f80-b45c-6af323ee502e">
 
 ## Getting Started
 
+### Prerequisites
+
+- Ensure that `npm` is installed on your device, or you can use a package manager like `nvm` to manage multiple versions of Node.js. [Read more](https://www.educative.io/answers/what-is-nvm)
+- Install pnpm, a package manager, to run the project.
+```bash
+npm install -g pnpm
+```
+
```

Comment:
`pnpm` can be installed in many ways and one need not necessarily need to have `npm`/`nvm` to install `pnpm`. Hence these need not be mentioned here, instead you can simply reference to the pnpm installation docs: https://pnpm.io/installation

Reply:
I thought most people have npm that's why we can use this method.
But your point is also right I will update this.